ComplementaryData/modelCuration/Metabolomics_newRxnProp.tsv

```diff
@@ -0,0 +1,22 @@
+RxnID	rev	RxnName	EC	rxnID_kegg	Source
+MNXR100390	0	ATP:D-Gluconate 6-phosphotransferase	2.7.1.12	R01737	rhea:19433
+MNXR123213	0	6-phosphogluconate phosphatase	3.1.3.-;3.1.3.2;3.1.3.29		metacyc:RXN0-5185
+MNXR110535	0	seed:rxn04605			seed:rxn04605
+MNXR110621	0	Carboxylic ester hydrolases	3.1.1.-;3.1.1.2		rhea:47384
+MNXR111769	1	phosphoglycerate dehydrogenase	1.1.1.95	R08198	rhea:13449
+MNXR134240	1	seed:rxn00681			seed:rxn00681
+MNXR95862	1	beta-alanine:2-oxoglutarate aminotransferase	2.6.1.19;2.6.1.55	R00908	rhea:30699
+MNXR101665	0	"3-Oxopropanoate:NAD+ oxidoreductase (decarboxylating, CoA-acetylating)"	1.2.1.-;1.2.1.18;1.2.1.27	R00705	rhea:22992

```

Comment:
please remove any `"` character from all 7 rxn names, as it unnecessary information to store